apps/web/.eslintrc.cjs

```diff
@@ -8,4 +8,19 @@ module.exports = {
       },
     },
   },
+  rules: {
+    "import/no-unused-modules": (() => {
+      // Trigger warning on CI only
```

Comment:
Could you please update the comment that this is only CI because it is slow?

Reply:
Yup! Made changes in [cbc6fe7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10741/commits/cbc6fe71a631e39d92193c161a470e8953a43792)